src/tbb/resource.rc

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) 2005-2022 Intel Corporation
```

Comment:
Please set copyright year in all resource files to 2022 only since this is the new file.